chart/prometheus-msteams/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
@Knappek @Flowkap how about we just use a new variable called apiVersion? This is how most of the stable helm charts did it. The default should be the current one or the new one.

Reply:
If only recent versions would support the "new one" ok. But honestly it's supported since a long time and removed in the last version. We're already on 1.17. without my proposed changes this one is undeployable. And I doubt it's used on older versions. Even if the older helm version of this chart works. For me this is already too much discussion. This was he only helm we use which was not 1.16 ready and I just wanted to help out a bit.